js/client-teambuilder.js

```diff
@@ -37,6 +37,7 @@
 
 			// details
 			'change .detailsform input': 'detailsChange',
+			'change .detailsform select': 'detailsChange',
```

Comment:
Technically the selects could be in a more optimized version of `detailsChange` (like `findDetailsChange`) because they dont need to update the UI, but I dont know that optimization matters.